tests/cypress/integration/actions_objects/case_37_object_make_copy.js

```diff
@@ -192,7 +192,11 @@ context('Object make a copy.', () => {
         it('Copy a shape with holding "Ctrl".', () => {
             const keyCodeC = 67;
             const keyCodeV = 86;
-            cy.get('#cvat_canvas_shape_18').trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated');
+            if (Cypress.isBrowser('!firefox')) {
+                cy.get('#cvat_canvas_shape_18').trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated');
+            } else {
+                cy.get('#cvat_canvas_shape_13').trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated');
+            }
```

Comment:
I don't think that this test depending on case_97 is a really good solution. Maybe it would be better to redesign it?

Reply:
I changed it to get and copy the first shape